### DISCUSSION TITLE
Is it possible to do a "if -> end, else -> workflow" loop?

### DISCUSSION THREAD

**Discussion by sandervspl** [langgenius/dify#5881] [Help]
![Screenshot 2024-07-02 at 14 03 52](https://github.com/langgenius/dify/assets/4492636/9a69b519-5bca-49e1-a002-1d8cadb76a55)

I want the `ELSE` to loop back to `Generate tips`. Is such a workflow possible?

**Comment by pengyuxiang1** [comment]
Here I am too, looking to develop a schematic diagram for the autoGPT framework, though unfortunately that feature isn't available right now.
